src/axom/sidre/interface/sidre_shroud.yaml

```diff
@@ -76,9 +77,10 @@ declarations:
 #  - decl: bool hasBuffer( IndexType idx ) const
   - decl:  Buffer * getBuffer( IndexType idx )
     fortran_generic:
-      idx:
-      -  int32_t
-      -  int64_t
+    - decl: (int32_t idx)
```

Comment:
In C, when an argument is int64_t and you pass in a 0 the compiler politely promotes it to the correct type. Since Fortran defaults to pass-by-reference this is not possible. Instead Shroud creates two function, one which accepts int32_t and one that accepts int64_t and allows them to be called by a generic name which will accept either argument. This allows the function to be called with a `0` instead of `0_C_INT64_t`.
Earlier version of Shroud only allowed one argument to be changed in the generic interface. Now multiple arguments can be generic.